pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1832,3 +1837,11 @@ type WorkerFailureMember struct {
 	PodName   string      `json:"podName,omitempty"`
 	CreatedAt metav1.Time `json:"createdAt,omitempty"`
 }
+
+// StorageVolume is TiKV storage information
+type StorageVolume struct {
+	Name             string  `json:"name,omitempty"`
+	StorageClassName *string `json:"storageClassName,omitempty"`
+	StorageSize      string  `json:"storageSize,omitempty"`
+	MountPath        string  `json:"mountPath,omitempty"`
```

Comment:
```suggestion
	Name             string  `json:"name"`
	StorageClassName *string `json:"storageClassName,omitempty"`
	StorageSize      string  `json:"storageSize"`
	MountPath        string  `json:"mountPath"`
```

Reply:
The above updated three fields are required for each config item, right?

Reply:
We should also add validation for the `StorageSize ` field to ensure it's a valid storage config.

Reply:
ok